New version: PrismAId v0.8.0

JuliaRegistrator opened the pull request:
- Registering package: PrismAId
- Repository: https://github.com/open-and-sustainable/prismaid
- Created by: @ricboer0
- Version: v0.8.0
- Commit: cef78ac33fffedb954a51b64dff218c81bf26ebe
- Reviewed by: @ricboer0
- Reference: https://github.com/open-and-sustainable/prismaid/issues/85#issuecomment-2850937718
- Description: Open Science AI Tools for Systematic, Protocol-Based Literature Reviews
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
There are no breaking changes. See the changelog.
`````
<!-- END RELEASE NOTES -->

<!-- daf33ad5e6d085e55119bc06703763ce03245391817b85c063d03a665c2dacb4c4cd54039bb567b9480011de0725b5c8866f6ef9aa89e7494ffce6ccf0dc92c17bb622d08ba6f19f2bfde974c10d064c24f83064d606f2be6c32e2b42b04758169a6844e3424aefcc2814db4e5395e04fd852c12c7ccb5e567174f2a67ce9752475074bb2522365fe8142177f981c8cbdcc189ae58d8fab5f08c8896b8cdddac3634096b006844845182cfeda7da8e058b6b2f63a7e9006037aacfb07cb76f0b -->

ricboer0 commented:
[merge approved]

@JuliaRegistrator register

Release notes:

There are no breaking changes. See the changelog.

JuliaRegistrator committed:
New version: PrismAId v0.8.0

UUID: 118a867e-9825-46ae-bed0-f8987ebdd75d
Repo: https://github.com/open-and-sustainable/prismaid.git
Tree: baf05fe745d5325490acdee2cd7cc9e2f7aa2c00

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

ricboer0 commented:
[merge approved]
